lambda/output.tf

```diff
@@ -13,6 +13,11 @@ output "function_role_arn" {
   value       = aws_iam_role.this.arn
 }
 
+output "function_version" {
+  description = "Version of the Lambda function."
+  value       = aws_iam_role.this.arn
```

Comment:
Should this be `aws_lambda_function.this.version`?

Reply:
Yes 🤦